shumai/tensor/tensor.ts

```diff
@@ -594,6 +595,10 @@ export class Tensor {
       ...arrayArg(stride, FFIType.i64)
     )
   }
+
+  softmax(axis: number): Tensor {
+    return op.softmax(this, axis)
+  }
```

Comment:
This file is likely to end up being pretty large in terms of lines of code, can you change this to a one liner as such `softmax(axis: number): Tensor => op.softmax(this, axis)` since it doesn't impact readability, but will make this 1 LOC vs 3 LOC.